airbyte-cdk/python/airbyte_cdk/sources/file_based/stream/default_file_based_stream.py

```diff
@@ -54,7 +55,7 @@ def state(self, value: MutableMapping[str, Any]) -> None:
         """State setter, accept state serialized by state getter."""
         self._cursor.set_initial_state(value)
 
-    @property
+    @property  # type: ignore # no suitable parent cursor type
```

Comment:
why is it not suitable / why can't the typing error be fixed?

Reply:
I looked at several of the `Cursor` offered by the IDE, and they didn't seem right. The only Cursor object that fixed the problem was the GCS Cursor, but this is a more generic file as it belongs to the CDK and not the specific GCS source.

Reply:
got it! Let's not mention the default at all then!

Reply:
what's the error exactly? Is the GCS cursor not an AbstractFileBasedCursor?

Reply:
The problem is that GCS cursor is not generic, as this file belongs to the CDK. Error is `error: Signature of "cursor" incompatible with supertype "Stream"  [override]`

Reply:
What’s the expected type from the parent class? Is it really a mismatch? If not, can the comment be more clear about why this is a mypy problem, not with the type